tests/visual/treeview.html

```diff
@@ -0,0 +1,173 @@
+<!doctype html>
+<html lang="en">
+<head>
+    <title>Treeview</title>
+
+    <link rel="stylesheet" href="../../packages/default/dist/all.css" />
+    <link rel="stylesheet" href="assets/styles.css" />
+    <script src="assets/scripts.js"></script>
+
+    <style>
+        #test-area {
+            max-width: 200px;
+            display: grid;
+            grid-gap: 25px 25px;
+            grid-template-columns: 1fr 1fr;
+            grid-template-rows: 1fr 1fr 1fr 1fr 1fr;
+        }
+    </style>
+</head>
+<body>
+    <div id="test-area">
+        <span></span>
+        <span>Example</span>
+
+        <span>Normal</span>
+        <section>
+            <kendo-treeview childrenfield="items" kendotreeviewexpandable="" kendotreeviewhierarchybinding="" kendotreeviewselectable="" textfield="text" class="k-widget k-treeview" role="tree" aria-multiselectable="false">
```

Comment:
Try to remove Angular-specific markup from the test case, as it isn't necessary for styling and will be confusing for future implementation of components for other frameworks.

Unused markup includes any directives, ng-* classes (apart from validation), unused `data-` attributes.